client/my-sites/design-preview/README.md

```diff
@@ -0,0 +1,12 @@
+# DesignPreview
+
+A wrapper for the always-available instance of `WebPreview` that's activated when `layoutFocus` is set to `preview`.
+
+The `DesignPreview` component can render raw markup to a preview and then apply customizations to that preview. This PR adds a data layer for the preview markup and the customizations.
```

Comment:
Actually, scratch that, since `WebPreview` is mentioned right in the previous sentence.